dbt/models/spatial/schema.yml

```diff
@@ -39,6 +39,31 @@ sources:
       - name: coordinated_care
         description: '{{ doc("table_coordinated_care") }}'
 
+      - name: corner
+        description: '{{ doc("table_corner") }}'
+        columns:
+          - name: pin10
+            description: '{{ doc("shared_column_pin10") }}'
```

Comment:
**[Question, non-blocking]** I would expect us to also have a `year` column on this table, are we missing docs for that or is it excluded for a reason?

Reply:
Fixed in 6c7a4c4f!